docs/Splashkit/Applications/Arcade Machines/Arcade Machine Setup/custom_splash.md

```diff
@@ -0,0 +1,48 @@
+# Update Arcade Machine Splash Screen
+
+## TL;DR
+Download the default splashkit svg file from the documentation images folder. Copy it to the Arcade Machine. Replace this file: `/opt/retropie/supplementary/emulationstation/resources/splash.svg` with the one you just downloaded. Reload emulationstation.
+
+## Background
+This guide explains how to replace the default arcade machine splash screen with a custom one. By default the user will see an "emulationstation" splash when running emulation station. This is a .svg format file which can be replaced with custom graphics at a power users discretion. 
+
+### Creating Replacement Splash
+Unfortunately the .svg file format is not well supported by basic image editing programs (GIMP, Paint, paint.net etc.). Inkscape is a high quality FOSS program which can work with .svg files. svg (Scalable Vector Graphics), like the name suggests, works with vectors, as opposed to "regular" image formats which typically use raster graphics of a set size. The advantage is that .svg files scale to any size without a loss in quality. The downside is that if your graphics are in a raster format, they must be converted first. 
+
+#### Converting a raster image to a scalable vector in Inkscape
+1. Open a new project in Inkscape
+1. Set your page dimensions to a 16:9 aspect ratio such as 1280x720 
+1. Drag & drop your raster image file (image.png) into the Inkscape window. Leave the default settings in the "image import" window and click "OK".
+1. Select the "Select & transform" tool (mouse icon on the left sidebar, hotkey:S) and click on the image. Then click on the drop down menu for Path > Trace bitmap
+1. The right side panel should change to a "Trace Bitmap" menu. Select Single Scan. From the Detection Mode drop down select Autotrace. Then click Apply down the bottom.
+1. Now toggle the pane selection from "Single Scan" to "Multicolor". In Detection mode select Colors. Then click Smooth, then Apply. 
+1. You should have three layers of images. The base raster image, a monochrome outline of the image and a colored vector image. Delete all except the colored vector image. 
+8. Repeat this process for any other image elements you wish to include in the splash. 
+
```

Comment:
There is some mistake here, can you make the number organize from 1 - 8.

Reply:
Good spot. I've fixed this.